packages/itinerary-body/src/styled.tsx

```diff
@@ -660,6 +660,18 @@ export const TransitAlert = styled.a`
   text-decoration: none;
 `;
 
+export const TransitAlertExternalLink = styled.a`
+  align-items: baseline;
+  display: flex;
+  gap: 5px;
+  margin-top: 0.5em;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
```

Comment:
Delete these default behaviors of `<a>` tags.

Reply:
By default there is an `underline`  on the `<a>` regardless of hover. I think I prefer it this way, where the underline only appears on hover, but I don't mind making it just the default behavior if you'd prefer!

Reply:
Oh okay that's right, then keep that CSS.